packages/server/src/fhir/operations/patientupdateaccounts.ts

```diff
@@ -0,0 +1,53 @@
+import { allOk, badRequest } from '@medplum/core';
+import { FhirRequest, FhirResponse } from '@medplum/fhir-router';
+import { Patient } from '@medplum/fhirtypes';
+import { getAuthenticatedContext } from '../../context';
+import { getPatientEverything } from './patienteverything';
+
+/**
+ * Handles the Patient $update-account operation.
+ * This operation updates the account reference for all resources in the patient compartment.
+ * The operation only updates resources from the getPatientEverything bundle that are not
+ * included in the patient's compartment, but are linked to the patient through a reference.
```

Comment:
I'm not quite sure what this means: the code looks like it __does__ operate on the resources from the Patient compartment